tests/functional/status.test.js

```diff
@@ -51,7 +52,7 @@ const {find} = require('lodash');
         });
 
         it('should report correct swarm_storage_usage', function () {
-            expect(this.crudModuleResponse.status).to.deep.include({'swarm_storage_usage': testParams.namespaceSize * testParams.numberOfNamespaces})
+            expect(this.crudModuleResponse.status).to.deep.include({'swarm_storage_usage': Math.floor(testParams.maxSwarmStorage / testParams.numberOfNamespaces)+ testParams.namespaceSize * testParams.numberOfNamespaces})
```

Comment:
Is this new piece related to the creation of a new DB on Line 19 w/ size `testParams.maxSwarmStorage/testParams.numberOfNamespaces`?

Reply:
I've revised the test logic, to make it much simpler. This is now just ``` testParams.numberOfNamespaces + testParams.namespaceSize ```

swarm_storage_usage is just the total of existing namespaces sizes.